com.kjxbyz.PicGuard.Pro.yml

```
@@ -0,0 +1,62 @@
+id: com.kjxbyz.PicGuard.Pro
+
+# https://docs.flatpak.org/en/latest/available-runtimes.html
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
```

this is not the latest.

```suggestion
runtime-version: '24.08'
```